### PR TITLE
Remove mentions of twitter

### PR DESCRIPTION
**Pull request by dawnpm**
Removes all mentions of twitter in documentation.
Mentions of twitter in release notes will remain.

## PR Checklist
- [Pending] Test Cloud.gov Pages preview for your branch for functionality - make sure all links and buttons work, javascript works as intended, and updates display as expected
  - Preview link: Previews not working currently
- [NA - no text changes] Conduct a HemingwayApp review of the content for plain language. Request plain language review in #helpwanted if needed before merging this PR
- [NA - no structural changes] Conduct a Lighthouse Audit via Chrome Developer Tools. Identify any issues that must be addressed before merging, and fix
- [ ] Ensure no non-public information is published in the commit messages

### CODE REVIEW

File: _admin-center/analytics/module-codes.md [hunk 1]
```diff
@@ -38,7 +38,6 @@ This page lists all the module codes and their full names.
 | QRTD  | Routed Queries |
 | SPEL  | Spelling Suggestions (Search.gov) |
 | SREL  | Related Searches |
-| TWEET | Tweets (Twitter) |
```
Comment (amanifarooque) [inline _admin-center/analytics/module-codes.md:41]: @dawnpm Previously, we've kept the deprecated codes in the table (and marked with an asterisk), in order to interpret prior analytics that might have that module code.

**Review by amanifarooque — COMMENTED** [review]
Looks good, but one open question around modules codes (whether we should delete or just mark as deprected).

File: _admin-center/analytics/module-codes.md (hunk 1 above)
Comment (dawnpm) [reply _admin-center/analytics/module-codes.md:41]: added back in, and added a note about why they're still there.

**Review by amanifarooque — APPROVED** [review]
LGTM

**Pull request merged**